### DISCUSSION TITLE
Is it possible to boost search scores at all?

### DISCUSSION THREAD

**Discussion by biot023** [meilisearch/product#548] [Feedback & Feature Proposal]
Hello. We're trying to balance relevance with an element of curation in our results but it's all seeming rather either/or.
If we place the `attribute` ranking rule above `favourite_count:desc`, for example, then relevancy appears to be king.
If we swap them around, though, then we see *much* less relevant (although much nicer) results.
Is there some way we could maybe have the best of both worlds by having something somewhere between the two?
I'm guessing not, from reading the docs, but I thought I'd ask to see if anybody has come up with a good way of doing this.
I suppose I'm thinking of some sort of multiplier rule for a search result 'score' but I don't think Meilisearch works that way, does it?
Anyway, I thought I'd ask in case someone had a really clever solution I could try applying. :)
Thanks for any and all (even negative) advice!
   Doug.

**Comment by gmourier** [comment]
Hi @biot023 👋

It could be related to these discussions

- https://github.com/meilisearch/product/discussions/272
- https://github.com/meilisearch/product/discussions/270
- https://github.com/meilisearch/product/discussions/379

You are right It's not possible right now with Meilisearch. 

How would you boost your documents, and on what criteria? How would you like to express this?

Thank you!

**Reply by fr3fou** [reply]
Any updates on this / plans for implementing something that helps this in the roadmap?

**Comment by herlon214** [comment]
Hi, I have a similar use case. I have a platform that help users to find events from artists (www.bandsnear.com if you want to check it out).
For me would be really good to have the ability of boosting some results. For instance, if the user search one artist name there's a chance of having two artists with the same name, but if I'm logged with my Spotify account the system can know which one is actually relevant to me (as a user).
So, this way I could boost some results in the search that will be more relevant to the user that is searching.

Would be interesting if there's a way to boost the results by specifying some filter (like ID = 123 OR ID = 456...) and a boost factor, idk like 10.

**Comment by macraig** [comment]
Hello everyone 👋

We just released a 🧪 prototype that allows **boosting results that match a custom filter**, and we'd love your feedback.

### How to use the prototype?
See https://www.notion.so/meilisearch/boostingFilter-API-usage-20aae06bc85e41dba828a90331a69f2c

⚠️ We do NOT recommend using this prototype in production. This is for test purposes only.

Feedback and bug reporting when using this prototype are encouraged! Thanks in advance for your involvement. It means a lot to us ❤️
